tests/test-extensions/some-extension/src/main/java/org/mule/test/some/extension/SomeExtension.java

```diff
@@ -7,22 +7,37 @@
 
 package org.mule.test.some.extension;
 
+import org.mule.runtime.extension.api.annotation.Configuration;
+import org.mule.runtime.extension.api.annotation.Configurations;
 import org.mule.runtime.extension.api.annotation.Export;
 import org.mule.runtime.extension.api.annotation.Extension;
 import org.mule.runtime.extension.api.annotation.Operations;
 import org.mule.runtime.extension.api.annotation.Sources;
 import org.mule.runtime.extension.api.annotation.connectivity.ConnectionProviders;
 import org.mule.runtime.extension.api.annotation.dsl.xml.Xml;
 import org.mule.runtime.extension.api.annotation.error.ErrorTypes;
+import org.mule.sdk.api.annotation.param.Optional;
+import org.mule.sdk.api.annotation.param.Parameter;
 import org.mule.test.heisenberg.extension.HeisenbergErrors;
 
+import java.time.ZonedDateTime;
+
 @Extension(name = "SomeExtension")
+@Configuration(name = "config")
```

Comment:
why is this needed?

Reply:
I added the name in order to reference the config on the parameter-value-resolver.xml file

Reply:
oh, it became required because you added an optional parameter to the config?